extensions/interactions/FractionInput/directives/fraction_input_interaction_directive.html

```diff
@@ -1,9 +1,11 @@
 <form ng-disabled="FractionInputForm.$invalid" ng-submit="submitAnswer(answer)" role="form" class="form-horizontal">
   <!-- This ng-form is needed for submitting answer on hitting enter. -->
-  <ng-form name="FractionInputForm">
-    <input type="text" name="answer" ng-model="answer" class="form-control"
-           ng-attr-placeholder="<[customPlaceholder !== '' ? customPlaceholder : (allowNonzeroIntegerPart ? ('I18N_INTERACTIONS_FRACTIONS_INPUT_PLACEHOLDER' | translate) : ('I18N_INTERACTIONS_FRACTIONS_INPUT_PLACEHOLDER_NO_INTEGER' | translate))]>">
-  </ng-form>
+  <fraction-input-interaction>
```

Comment:
Ah, when I suggested adding a fraction-input-interaction to the custom class, I had assumed that this directive was already wrapped in a fraction-input-interaction tag in the DOM. Is this not the case?

If it is the case, we don't need this line. If it is not the case, then let's just add a .fraction-input-form class to the ng-form and remove this line anyway.

Reply:
Done.